packages/commonwealth/server/bindings/bootstrap.ts

```diff
@@ -118,21 +128,47 @@ export async function bootstrapRelayer(
 }
 
 export function bootstrapContestRolloverLoop() {
-  log.info('Starting rollover loop');
+  const cronFrequency = '* * * * *'; // every minute
+
+  log.info(`Starting rollover cron job (${cronFrequency})`);
+
+  CronJob.from({
+    cronTime: cronFrequency,
+    onTick: async () => {
+      try {
+        await handleEvent(ContestWorker(), {
+          name: 'ContestRolloverTimerTicked',
+          payload: {},
+        });
+      } catch (err) {
+        log.error(err);
+      }
+    },
+    start: true,
+  });
+}
 
-  const loop = async () => {
-    try {
-      await handleEvent(ContestWorker(), {
-        name: 'ContestRolloverTimerTicked',
-        payload: {},
-      });
-    } catch (err) {
-      log.error(err);
-    }
-  };
+export function bootstrapCommunityIndexerLoop() {
+  const cronFrequency = config.COMMUNITY_INDEXER.CRON!;
+  if (!cronFrequency) {
+    log.warn('Skipping community indexer cron job');
+    return;
+  }
 
-  // TODO: move to external service triggered via scheduler?
-  setInterval(() => {
-    loop().catch(console.error);
-  }, 1_000 * 60);
+  log.info(`Starting community cron job (${cronFrequency})`);
+
+  CronJob.from({
```

Comment:
We can use Graphile here instead of CronJob. I'm fine with merging as is if you prefer and then fast following.